spec/requests/api/allocations_controller_index_spec.rb

```diff
@@ -88,11 +88,15 @@
           per_page: 5,
           total_pages: 2,
           total_objects: 6,
-          links: {
-            first: '/api/v1/allocations?page=1',
-            last: '/api/v1/allocations?page=2',
-            next: '/api/v1/allocations?page=2',
-          },
+        }
+      end
+      let(:pagination_links) do
+        {
+          self: 'http://www.example.com/api/v1/allocations?page=1&per_page=5',
```

Comment:
Are we okay with fully qualified over relative links? I presume this has been discussed somewhere with Dom et al. Probably I missed it

Reply:
These were previously fully qualified with the AMS implementation (don't confuse them with the relative links that were in the metadata section, these are in the top level of the document). Also the links were previously broken with erroneous `page[number]&page[per_page]` parameters - thanks AMS - so this PR fixes that. The front end doesn't actually look at the content of the links anyway, only for presence of those e.g. https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/services/move.js#L33

The JSONAPI spec seems to suggest full URL's in the example but it's not made explicit as far as I can tell: https://jsonapi.org/format/#document-links

TL:DR; Yes 😄

Reply:
Thanks Andrew. Very clear.